x-pack/test_serverless/api_integration/config.base.ts

```diff
@@ -28,6 +28,7 @@ export function createTestConfig(options: CreateTestConfigOptions) {
           `--xpack.alerting.enableFrameworkAlerts=true`,
           '--xpack.observability.unsafe.thresholdRule.enabled=true',
           '--server.publicBaseUrl=https://localhost:5601',
+          '--xpack.task_manager.requeue_invalid_tasks.enabled=true',
```

Comment:
Since this is now part of `config/serverless.yml`, I think we don't need it here anymore.

Reply:
Does `config/serverless.yml` settings work when running serverless tests locally?

Reply:
For local and CI tests, we're starting Kibana with the `--serverless=<project>` flag, which also includes loading the `config/serverless.yml` settings.

Reply:
Awesome, thanks for the explanation. 
Removed the config.